Tests/SkillFunctionalTests/Utils/README.md

```diff
@@ -2,21 +2,59 @@
-<p align="center">
-  <img src="https://user-images.githubusercontent.com/54330145/124788742-0756ab00-df20-11eb-8eea-8f54c07708f1.png" />
-</p>
+### Inputs
+
+| Input                   | Description                                                                                                                                                   | Condition | Default              | Example                                                           |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- | -------------------- | ----------------------------------------------------------------- |
+| ResourceGroup           | The Name for the specific Resource Group where the resources are deployed, For each specific language will concatenate (DotNet, JS and Python).               | Required  | BFFN                 | "bffnbots"                                                        |
+| ResourceSuffix          | The Suffix used to concatenate at the end of the Resource Name followed up by the ResourceSuffixSeparator.                                                    | Required  |                      | "microsoft-396"                                                   |
+| Subscription            | The Name or Id of the Subscription where the resources are located.                                                                                           | Optional  | Current Subscription | "00000000-0000-0000-0000-000000000000" or "bffnbots-subscription" |
+| ResourceSuffixSeparator | The separator used for the Suffix to split the Resource Name from the ResourceSuffix. <br> **Note:** _Only available when providing it through `parameters`_. | Optional  | -                    | "" or "-microsoft-"                                               |
+
+### The inputs can be provided as `prompts` or `parameters`
+
+1. When using `prompts`, it will ask for the `required` inputs to be provided. When no input is entered, it will use the `default (...)` value instead. For more information about, see [Inputs][inputs].
+
+   ![prompts][prompts]
+
+2. When using `parameters`, all listed inputs can be provided before executing the script. When no input is entered, it will use the `default` value instead. For more information, see [Inputs][inputs].
+
+   ![parameters][parameters]
+
+### Result
+
+After providing the desired [Inputs][inputs], the script will start looking for the Bot Resources, followed up by gathering each DirectLine Secret from the Azure Bot Resource and set it in the `appsettings.Development.json`. Moreover, all steps will be shown in the terminal as well as the result with each DirectLine Secret set for each Bot.
+
+![sample][sample]
+
+> **Note:** When not `appsettings.Development.json` file is found, it will proceed by generating a copy from the `appsettings.json` baseline file.

```

Comment:
```suggestion
> **Note:** When no `appsettings.Development.json` file is found, it will proceed by generating a copy from the `appsettings.json` baseline file.

```